### PR TITLE
add bootstrap-star-rating v4.0.0

### PR DESCRIPTION
**Pull request by maruilian11**
**git repo url:** https://github.com/kartik-v/bootstrap-star-rating
**Watch 49
  Star 466
 Fork 179**
@Amomo could u help me check this PR for #6558 ? thank you!

Checklist for **Pull request** or **lib adding request issue** follows the conventions.

# Essential checklist
 * [ ] I'm the author of this library
   * [ ] I would like to add link to the page of this library on CDNJS on website / readme
 * [x] This lib was not found on cdnjs repo
 * [x] No already exist / duplicated issue and PR
 * [x] The lib has notable popularity
   * [x] More than 100 [Stars / Watchers / Forks] on [GitHub / Bitbucket]
   * [ ] More than 500 downloads stats per month on npm registry
 * [x] Project has public repository on famous online hosting platform (or been hosted on npm)

# Auto-update checklist
 * [ ] Has valid tags for each versions (for git auto-update) https://github.com/kartik-v/bootstrap-star-rating/issues/105
 * [x] Auto-update setup
 * [x] Auto-update target/source is valid.
 * [x] Auto-update filemap is correct.

# Git commit checklist
 * [x] The first line of commit message is less then 50 chars, be clean and clear, easy to understand.
 * [x] The parent of the commit(s) in the PR is not old than 3 days.
 * [x] Pull request is sending from a non-master branch with meaningful name.
 * [x] Separate unrelated changes into different commits.
 * [x] Use rebase to squash/fixup dummy/unnecessary commits into only one commit.
 * [x] Close corresponding issue in commit message
 * [x] Mention related issue(s), people in commit message, comment.

### CODE REVIEW

File: ajax/libs/bootstrap-star-rating/package.json [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    "bootstrap": "~3"
+  },
+  "license": "BSD-3-Clause",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/kartik-v/bootstrap-star-rating.git",
+    "basePath": "",
+    "files": [
+      "css/star-rating.css",
+      "css/theme-krajee.css",
```
Comment (Amomo) [inline ajax/libs/bootstrap-star-rating/package.json]: Is it `"css/theme-krajee*.css"`?

File: ajax/libs/bootstrap-star-rating/package.json [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+      "img/loading.gif",
+      "js/star-rating.js",
+      "js/star-rating_locale.js"
```
Comment (Amomo) [inline ajax/libs/bootstrap-star-rating/package.json]: `"js/star-rating_locale*.js"`?

**Comment by maruilian11** [comment]
@Amomo i have added `*` for auto-update config, could you help me check it again? thanks!

**Comment by PeterDaveHello** [comment]
Why do we need `bootstrap-star-rating/4.0.0/js/star-rating_locale_LANG.js`?

**Comment by maruilian11** [comment]
https://github.com/cdnjs/cdnjs/pull/7408

**Pull request closed without merging**